app/policies/hcb_code_policy.rb

```diff
@@ -2,7 +2,7 @@
 
 class HcbCodePolicy < ApplicationPolicy
   def show?
-    user&.auditor? || present_in_events? || (record.stripe_cardholder.present? && record.stripe_cardholder.user == user)
+    user&.auditor? || present_in_events? || cardholder?
```

Comment:
Cardholder doesn't seem to be defined in `HcbCodePolicy`, it's defined here: https://github.com/thedev132/hcb/blob/04aaf858e636c316bc2755be639d7579672486f2/app/policies/stripe_card_policy.rb#L62 in `StripeCardPolicy`

Reply:
ohhhhh I see.